### DISCUSSION TITLE
How to remove spaces from selected option?

### DISCUSSION THREAD

**Discussion by Marcelogiovane** [sa-si-dev/virtual-select#216] [Q&A] [answered]
The original option has spaces that must be eliminated in the display.

**Comment by sa-si-dev** [comment]
@Marcelogiovane Please share a sample code or screenshot of it to understand your question better.

**Comment by Marcelogiovane** [comment]
To indent option without optiongroup there are spaces on the left that must be removed after selection.
[Captura de tela 3.pdf](https://github.com/sa-si-dev/virtual-select/files/10344536/Captura.de.tela.3.pdf)

**Comment by sa-si-dev** [comment, marked as answer]
@Marcelogiovane use [labelRenderer](https://sa-si-dev.github.io/virtual-select/#/examples?id=add-imageicon) to add space in the options label.
